logging/logging.go

```diff
@@ -632,7 +632,7 @@ func jsonValueToStructValue(v interface{}) *structpb.Value {
 // and will block, it is intended primarily for debugging or critical errors.
 // Prefer Log for most uses.
 func (l *Logger) LogSync(ctx context.Context, e Entry) error {
-	if e.Severity < l.minimalLoggedSeverity {
+	if e.Severity != Default && e.Severity < l.minimalLoggedSeverity {
```

Comment:
nit: a comment here would also be useful. Something like `Ignore logs with severity less than minimalLoggedSeverity. Default severity logs are always written`